### DISCUSSION TITLE
Ban application

### DISCUSSION THREAD

**Discussion by LexiCodesOfficial** [pterodactyl/panel#4064] [General Community]
I Was Hacked A While Back And Got Banned Could I Get UnBanned?

**Comment by iamkubi** [comment]
You'd need to at the very least include your Discord account name.  Preferably ID.

**Reply by LexiCodesOfficial** [reply]
468451791613198337

**Reply by iamkubi** [reply]
Unbanned.  Don't scan random codes.

**Reply by LexiCodesOfficial** [reply]
Okay Thankyou!
